default/scripting/buildings/CONC_CAMP.focs.txt

```diff
@@ -25,7 +25,7 @@ BuildingType
             priority = [[TARGET_AFTER_SCALING_PRIORITY]]
             effects = SetTargetIndustry value = Value + Target.Population
                         * (NamedReal name = "BLD_CONC_CAMP_TARGET_INDUSTRY_PERPOP"
-                                     value = 10 * [[INDUSTRY_PER_POP]])
+                                     value = 3.75 * [[INDUSTRY_PER_POP]])
```

Comment:
A bit confusing how this relates to the description and .75 vs 2x pop...

Reply:
I'm not sure I see your confusion.
INDUSTRY_PER_POP == 0.2
3.75 * 0.2 == 0.75
10 * 0.2 == 2.0

No body ever changes INDUSTRY_PER_POP, RESEARCH_PER_POP...
I've been thinking on changing their values to 1.0 and update the factors accordingly (so in this case I would set 0.75*[[INDUSTRY_PER_POP]]).